### ISSUE TITLE
SPFx React Slide Swiper: Unable to swipe on cards in mobile view

### ISSUE THREAD

**Issue by ishanisuchak** [SharePoint/sp-dev-fx-webparts#1003]
Category

- [ ] Question
- [x] Bug
- [ ] Enhancement

## Authors

**@VelinGeorgiev**

## Expected or Desired Behavior

Users should be able to swipe left and right on the slider cards to go to the previous or next card.

## Observed Behavior

I am able to swipe when in desktop view. Even, in a mobile browser, on switching to desktop view, users are able to swipe.
But, in mobile view, the swipe functionality is completely disabled and we can change the slide only by using the navigation arrows or dots.

## Steps to Reproduce

1. Build and package the solution
2. Add the app to a SharePoint modern site
3. Add the webpart to any site page and configure list with data
4. Open the site page in a phone

(I have tested in chrome for iOS and Android and am able to see the issue in both.)

**Comment by VelinGeorgiev** [comment]
I will look at this asap.

**Comment by ishanisuchak** [comment]
I had created an SPFx webpart using the react-slick package and was able to see the same issue for mobile browsers. However, I was able to fix the issue for now, by adding the'ms-Overlay' class to HTML elements that i want to be swipeable.
I found on debugging that MS is blocking the touchMove event for all elements that do not have this class associated to them.
This might solve the issue for this webpart too.
For now, it looks like a blocker from MS, so closing this issue.
